scripts/xgo_run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+COMMAND=$@
+if [ -z "$COMMAND" ]; then
+    printf "\e[0;31m%s\e[0m\n" "Missing command!"
+    exit 1
+fi
+
+docker run -it --rm \
+    -v `pwd`:/go-src-root/ \
+    -v `pwd`:/go/src/github.com/mysteriumnetwork/go-openvpn \
+    -w /go/src/github.com/mysteriumnetwork/go-openvpn \
+    --entrypoint "/bin/bash" \
+    mysteriumnetwork/xgo:1.11 -c "${COMMAND}"
```

Comment:
We prob should have xgo:myst tag we support, so we should not go over and over all sources changing every time we bump version up. Created a ticket for this. #414

Reply:
fair point